src/main/java/frc/robot/commands/drivetrain/SimpleDriveDistance.java

```diff
@@ -0,0 +1,51 @@
+package frc.robot.commands.drivetrain;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.DrivetrainSubsystem;
+
+/*
+ * Drives forward for the specified distance.
+ */
+public class SimpleDriveDistance extends CommandBase {
+
+    private final DrivetrainSubsystem m_drivetrain;
+    private final double m_distance;
+    private final double m_threshold;
+    private final double m_speed;
+    private double m_last_distance;
+
+    public SimpleDriveDistance(DrivetrainSubsystem drivetrain, double distance, double threshold, double speed) {
+        m_drivetrain = drivetrain;
+        m_distance = distance;
+        m_threshold = threshold;
+        m_speed = speed;
+    }
+
+    @Override
+    public void initialize() {
+        m_last_distance = m_drivetrain.getDistance();
+    }
+
+    @Override
+    public void execute() {
+        if (getError() > 0) {
+            m_drivetrain.arcadeDrive(m_speed, 0.0);
+        } else {
+            m_drivetrain.arcadeDrive(-m_speed, 0.0);
```

Comment:
If the error is exactly 0 (zero) should it continue driving in the execute? I think that if it seems we are exactly on the target distance (e.g. getError() returns zero, as in there is no more distance to travel before we reach the target) it doesn't make sense to drive backwards at all. I think this line should be contained within an else-if block which runs if the error is less than zero.

Reply:
So this is actually fine because isFinished will see that the error is less than the threshold and stop the drivetrain

For clarity, I can adjust it though

Reply:
Added an else if